src/components/NavBarButtons/NavBarButtons.js

```diff
@@ -1,14 +1,14 @@
 import { Link } from "react-router-dom";
-import { ButtonContainer, Button } from "./NavBarButtons.styles";
+import { ButtonContainer, Button, ButtonText, styledLink } from "./NavBarButtons.styles";
 
 const NavBarButtons = (props) => {
   return (
     <ButtonContainer>
-      <Link to="/">
-        <Button>Coins</Button>
+      <Link to="/" style={styledLink}>
```

Comment:
you can create a styled link like this

const StyledLink = styled(Link)`
  your styles here
`

and then use like

<StyledLink  to='/some-path' />